libp2p/protocols/pubsub/gossipsub.nim

```diff
@@ -318,7 +318,7 @@ method onNewPeer(g: GossipSub, peer: PubSubPeer) =
     discard
 
 proc grafted(g: GossipSub, p: PubSubPeer, topic: string) =
-  g.peerStats.withValue(p, stats) do:
+  g.peerStats.withValue(p, stats):
```

Comment:
Interesting, does removing the `do` has any noticeable effect here?

Reply:
`do` is needed when passing two blocks to withValue - otherwise it makes no difference (this is likely a copy-paste leftover)